packages/vscode-graphql/package.json

```
@@ -179,7 +179,7 @@
     "ovsx": "0.8.3"
   },
   "dependencies": {
-    "graphql": "^16.8.1",
+    "graphql": "^16.8.1 || ^17.0.0-alpha.2",
```

```suggestion
    "graphql": "^16.8.1 || ^17.0.0-alpha.5",
```
to match other updates in monorepo